src/v/raft/vote_stm.cc

```diff
@@ -49,9 +50,13 @@ vote_stm::~vote_stm() {
       "Must call vote_stm::wait()");
 }
 ss::future<result<vote_reply>> vote_stm::do_dispatch_one(vnode n) {
-    auto tout = _ptr->_jit.base_duration();
+    auto tout = _ptr->_jit.base_duration() * 3;
```

Comment:
didn't understand the "* 3" part, whats special about 3?

Reply:
i just use a timeout that is 3 time greater than the election timeout, not to introduce an additional parameter.

Reply:
@mmaslankaprv wondeirng if  `jit` should have a method `base_duration_with_multiplier(3)` or smth like that.